marm-mcp-server/marm_mcp_server/core/memory.py

```
@@ -723,11 +759,11 @@ async def recall_similar(
                         }
                     )
 
-                return results
+                return _wrap(results, scan_truncated)
 
         except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)